sdv/metadata/table.py

```diff
@@ -221,7 +249,7 @@ def __init__(self, name=None, field_names=None, field_types=None, field_transfor
         self._sequence_index = sequence_index
         self._entity_columns = entity_columns or []
         self._context_columns = context_columns or []
-        self._constraints = constraints or []
+        self._constraints = self._prepare_constraints(self._constraints)
```

Comment:
Hey! This should the `(constraints)` now, since `self._constraints` is still unassigned.

Reply:
yeah, tests caught it thankfully